### ISSUE TITLE
[Feature] Provide first steps recommendation on the first time running with berry

### ISSUE THREAD

**Issue by orta** [yarnpkg/berry#596]
**Describe the user story**

I have just started looking at berry-ing a project, I have questions like:

- should I delete my `node_modules` folder (I'm like 90% on yes) 
- should I delete my `yarn.lock` (probably but with less confidence)
- there are now 4 new root files and folders ( `.yarn`, `.yarnrc`, `.yarnrc.yml`, `.pnp.js`) a one liner about why each exist would be 👍 

**Describe the solution you'd like**

When berry sees: yarn.lock & node_modules and is creating `yarnrc.yml` for the first time. It can pretty safely assume a first time migration and finish the initial `yarn install` with some a congrats like:

```sh
Thanks for trying out yarn 2 🎉

Your next steps:

- Remove your node_modules with `rm -rf */node_modules` (or whatever it takes to recursively delete)
- Remove your yarn.lock from the repo
- We've added the following new files:
 - .yarn - keeps your modules in a zip file, you want to keep this git
 - pnp.js - helps tools understand where modules are
 - `yarnrc.yml` - the config for yarn v2
 - `.yarnrc` - the config for yarn v1 to tell it to use v2

You can find out more in https://next.yarnpkg.com/advanced/migration

```

**Describe the drawbacks of your solution**

It's a bit verbose, and you _could_ argue that the `.yarnrc` was set from `yarn policies set-version v2` which isn't this command

**Describe alternatives you've considered**

More docs in the migration guide?

**Comment by arcanis** [comment]
> should I delete my node_modules folder (I'm like 90% on yes)

Yup - we print a warning during linking if we notice that there's a lingering `node_modules`. The warning error code points to [here](https://next.yarnpkg.com/advanced/error-codes#yn0031---dangerous_node_modules), which I should edit to include a command to remove them in one go!

> should I delete my yarn.lock (probably but with less confidence)

Nope - Yarn will automatically migrate the lockfile and reuse the versions that are already stored inside. We'll still need to make network requests to retrieve the necessary information that weren't part of the v1 format, but apart from that keeping the lockfile is advised.

> there are now 4 new root files and folders ( .yarn, .yarnrc, .yarnrc.yml, .pnp.js) a one liner about why each exist would be 👍

> It's a bit verbose, and you _could_ argue that the `.yarnrc` was set from `yarn policies set-version v2` which isn't this command

I like the idea of printing a "welcome message". Maybe what we could do would be to print such a message when running `yarn policies set-version v2`. This way after the stable release we could start putting things like the changelog, etc 🤔
